frontend/components/ClubPage/MemberCard.js

```diff
@@ -0,0 +1,49 @@
+import React from 'react'
+import s from 'styled-components'
+import { BORDER } from '../../constants/colors'
+
+const Card = s.div`
+  text-align: center;
+  width: 24%;
```

Comment:
Does this work on mobile devices? I recommend using `<Row />` and `<Col />` components we have in shared to make sure this looks good on all devices.

See example here: https://github.com/pennlabs/pennlabs.org/blob/master/src/components/Home/Stats.tsx#L58-L69